src/nodelets/BosonCamera.cpp

```diff
@@ -474,6 +475,19 @@ void BosonCamera::captureAndPublish(const ros::TimerEvent &evt)
       ci->header.stamp = pub_image_8->header.stamp;
       image_pub_8.publish(pub_image_8, ci);
 
+      // 8bit image (auto range)
+      double min, max;
+      minMaxLoc(thermal8_linear, &min, &max);
+      Mat thermal8_norm = (thermal8_linear - min) * (255 - 0) / (max - min);;
```

Comment:
Could you handle the case where `max == min`?

Reply:
Thank you! Add checking 0 devide.